alphaverse/expr/transformational_operator.py

```diff
@@ -203,3 +206,32 @@ def evaluate(self, store: Store) -> pl.Series:
     @property
     def children(self) -> List[Expr]:
         return [self._child, self._condition, self._destination]
+
+
+class Winsorize(VectorExpr):
+    """팩터 값의 상한값과 하한값을 각각 전체의 upper, lower 분위수에 해당하는 값으로 대체합니다."""
+
+    def __init__(
+        self,
+        child: VectorExpr,
+        lower: ScalarExpr[float],
+        upper: ScalarExpr[float],
+    ) -> None:
+        self._child = child
+        self._lower = lower
+        self._upper = upper
+
+    def evaluate(self, store: Store) -> pl.Series:
+        child = self._child.evaluate(store)
+        lower = self._lower.evaluate(store)
+        upper = self._upper.evaluate(store)
+        _child = child.drop_nans().drop_nulls().unique()
```

Comment:
```suggestion
        _child = child.drop_nans().drop_nulls()
```
unique만 걸러내는 로직은 제외하는게 맞는 듯 합니다..! : )

Reply:
[12f2776](https://github.com/alphaprime-dev/alphaverse/pull/53/commits/12f27764d41b2563bc38031a5964bd52d372c6d7)